eo-maven-plugin/src/main/java/org/eolang/maven/HashOfTag.java

```diff
@@ -34,6 +34,9 @@
 /**
  * Hash of tag.
  * @since 0.26
+ * @todo #1062:30min Make HashOfTag testable without accessing web.
```

Comment:
@mximp this problem doesn't affect the work of this particular branch, right? in this case, just submit it as a new ticket to the repo, no need to create a puzzle